services/ansibler/server/ansible-playbooks/wireguard/templates/wg.conf.j2

```diff
@@ -7,7 +7,7 @@ ListenPort = {{ wg_listen_port }}
 [Peer]
 PublicKey = {{ hostvars[host].publickey.stdout }}
 Endpoint = {{ hostvars[host].ansible_host }}:{{ wg_listen_port }}
-AllowedIps = {{ hostvars[host].private_ip }}/32
+AllowedIps = {{ hostvars[host].private_ip }}/{{ netmask}}
```

Comment:
When discussed during the last grooming, I was wondering how could've `/32` ever worked.

> The keyword allowed-ips is a list of addresses that will get routed to the peer.

Ref: https://wiki.archlinux.org/title/WireGuard

Reply:
I was thinking the same but turns out the `/32` is needed (I misunderstood. the CIDR notation)

The `/32` means only a single IP address whereas anything lower than that means a range of IP addresses,
And if we have multiple clients with the same network `/32` is needed as otherwise wireguard couldn't resolve to which client to send the packets

more: 
https://www.reddit.com/r/WireGuard/comments/enh9xa/help_on_24_and_32_when_using_as_a_vpn_server/
https://www.wireguard.com/#cryptokey-routing